Sources/App/Commands/Ingest.swift

```diff
@@ -224,3 +249,12 @@ private extension Github.Metadata.Repository {
     var repositoryOwner: String? { owner.login }
     var repositoryName: String? { name }
 }
+
+private extension URL {
+    var normalizedParent: Self? {
```

Comment:
```suggestion
    var normalized: Self? {
```

as this is for URLs generally.

And I think we should drop the `private` and put it in a new file `URL+ext.swift` in `Core/Extensions`.